### PR TITLE
Fix upgrade instructions for docker load option

### PR DESCRIPTION
**Pull request by schmichael**

### CODE REVIEW

File: website/source/docs/upgrade/upgrade-specific.html.md [hunk 1]
```diff
@@ -25,8 +25,24 @@ images to load, in 0.5.5 it has been changed to a single string. No
 functionality was changed. Even if more than one item was specificed prior to
 0.5.5 only the first item was used.
 
-If you have jobs that use the `load` option first upgrade all nodes to 0.5.5,
-then resubmit those jobs with `load` as a single string.
+To do a zero-downtime deploy with jobs that use the `load` option:
+
+* Upgrade to servers to 0.5.5 or later.
```
Comment (dadgar) [inline website/source/docs/upgrade/upgrade-specific.html.md]: "Upgrade to servers to 0.5.5 or later." -> "Upgrade servers to version 0.5.5 or later."

File: website/source/docs/upgrade/upgrade-specific.html.md [hunk 2]
```diff
@@ -25,8 +25,24 @@ images to load, in 0.5.5 it has been changed to a single string. No
...
+
+* Deploy new client nodes on the same version.
```
Comment (dadgar) [inline website/source/docs/upgrade/upgrade-specific.html.md]: Same version as the servers.

File: website/source/docs/upgrade/upgrade-specific.html.md [hunk 3]
```diff
@@ -25,8 +25,24 @@ images to load, in 0.5.5 it has been changed to a single string. No
...
+* Resubmit jobs with the `load` option fixed and a constraint to only run on
+  version 0.5.5 or later:
+
+```hcl
+    constraint {
+      attribute = "nomad.version"
+      operator  = "version"
+      value     = ">= 0.5.5"
+    }
+```
```
Comment (phinze) [inline website/source/docs/upgrade/upgrade-specific.html.md]: @schmichael after we ran with this we ended up realizing the syntax is actually a bit different - here's the working snippet:

```hcl
  constraint {
    attribute = "${attr.nomad.version}"
    operator  = "version"
    value     = ">= 0.5.5"
  }
```

Comment (schmichael) [inline website/source/docs/upgrade/upgrade-specific.html.md]: Ack, thanks. Fixing.

**Pull request merged**